tests/snappi_tests/multidut/pfc/files/lossless_response_to_external_pause_storms_helper.py

```diff
@@ -126,6 +127,21 @@ def run_lossless_response_to_external_pause_storms_test(api,
                                                    all_flow_names=all_flow_names,
                                                    exp_dur_sec=DATA_FLOW_DURATION_SEC + DATA_FLOW_DELAY_SEC,
                                                    snappi_extra_params=snappi_extra_params)
+    interface_stats = get_interface_stats
```

Comment:
No arguments needed ? the definition is :
def get_interface_stats(duthost, port):

Reply:
arguments is being passed in line 134